src/renderer/root/components/AuthenticatedLayout/Navigation/Navigation.js

```diff
@@ -20,33 +20,39 @@ export default function Navigation(props) {
   return (
     <nav className={classNames(styles.navigation, props.className)}>
       <ul className={styles.group}>
+        {/* <li>
+          <Tooltip overlay="DApps">
+            <div>
+              <TabLink id="dapps" target={DAPPS}>
+                <DAppsIcon aria-label="dapps" />
+              </TabLink>
+            </div>
+          </Tooltip>
+        </li> */}
```

Comment:
comments can be removed?

Reply:
I generally agree with not leaving commented out code in a repo.  I decided against it here simply because we'll be adding it back in the near future.

Reply:
Ah right lol 😄 Didn't looked at WHAT was commented out